suite-native/app/src/navigation/RootStackNavigator.tsx

```diff
@@ -35,40 +43,51 @@ export const RootStackNavigator = () => {
         return RootStackRoutes.Onboarding;
     };
 
+    // Take screenshot on navigation state change so it can be used for biometrics and other overlays later.
+    useEffect(() => {
+        const removeStateSubscription = navigation.addListener('state', () => {
+            takeScreenshot(screenshotViewRef);
+        });
+
+        return removeStateSubscription;
```

Comment:
stateSubscription and then return stateSubscription.removeListener, no?

Reply:
Not really. The react-navigation event listeners do not return an object, but rather the unsubscribe function right away. https://reactnavigation.org/docs/navigation-events/#navigationaddlistener

Reply:
Ah, cool